src/frontend/planner_test/tests/testdata/output/append_only.yaml

```diff
@@ -7,7 +7,7 @@
     └─StreamProject { exprs: [t1.v1, max(t1.v2)] }
       └─StreamHashAgg [append_only] { group_key: [t1.v1], aggs: [max(t1.v2), count] }
         └─StreamExchange { dist: HashShard(t1.v1) }
-          └─StreamTableScan { table: t1, columns: [t1.v1, t1.v2, t1._row_id], stream_scan_type: Backfill, pk: [t1._row_id], dist: UpstreamHashShard(t1._row_id) }
```

Comment:
An alternative is to rename...

- `Backfill` to `NoShuffleBackfill`
- `ArrangementBackfill` to `Backfill`

so there's no change on the plans. 😄